index.js

```diff
@@ -1,4 +1,5 @@
 var React = require('react');
+var assign = require('react/lib/Object.assign');
```

Comment:
Please don't use react/lib/* (all of which we consider private); a good choice for this case is https://www.npmjs.com/package/object-assign.